ui/app/src/Views/NavBar/Views.elm

```diff
@@ -9,12 +9,27 @@ import Views.NavBar.Types exposing (Tab, alertsTab, silencesTab, statusTab, none
 navBar : Route -> Html msg
 navBar currentRoute =
     header
-        [ class "navbar navbar-toggleable-md navbar-light "
-        , style [ ( "margin-bottom", "3rem" ), ( "border-bottom", "1px solid rgba(0,0,0,.15)" ) ]
+        [ class "navbar navbar-toggleable-md navbar-light bg-faded mb-5 pt-3 pb-3"
+        , style [ ( "border-bottom", "1px solid rgba(0, 0, 0, .125)" ) ]
         ]
         [ nav [ class "container" ]
-            [ a [ class "navbar-brand", href "#" ] [ text "AlertManager" ]
+            [ a [ class "navbar-brand", href "#" ] [ text "Alertmanager" ]
             , ul [ class "navbar-nav" ] (navBarItems currentRoute)
+            , case currentRoute of
+                SilenceFormEditRoute _ ->
+                    text ""
+
+                SilenceFormNewRoute _ ->
+                    text ""
+
+                _ ->
```

Comment:
I thought about this:

* Silence is the only resource that is manageable from the UI
* Silence creation is one of the most important functions
* Navigating to the silences page just so that the button appears in the header feels strange

Why not have it as a shortcut?